slurm/nccl-test-distributed-a100-64-enroot.slurm

```diff
@@ -15,9 +15,52 @@ export NCCL_SOCKET_IFNAME=eth0
 export NCCL_IB_HCA=ibp
 export UCX_NET_DEVICES=ibp0:1,ibp1:1,ibp2:1,ibp3:1
 
+# Define nccl version we will test 
+# See https://github.com/coreweave/nccl-tests/pkgs/container/nccl-tests for available tags.
+
+nccl_version="12.9.1-devel-ubuntu22.04-nccl2.27.5-1-d5a135d"
+
+# Create a directory to store container images. You can change this 
+# or use an existing directory below. 
+
+CONTAINER_DIR=`pwd`/images
+mkdir -p $CONTAINER_DIR
+
+fstype=`df -T $CONTAINER_DIR| awk 'NR==2 {print $2}'`
+if [ "$fstype" != "nfs" ] ; then
+  echo "You must specify a container directory that is mounted on all cluster nodes."
+  exit 1
+fi
```

Comment:
Edit to use more precise utilities, quote variable expansions, and print errors to stderr.
```suggestion
CONTAINER_DIR="$(realpath -s images)"
mkdir -p "$CONTAINER_DIR"

fstype="$(stat '-fc%T' "$CONTAINER_DIR")"
if [ "$fstype" != "nfs" ] ; then
  echo 'You must specify a container directory that is mounted on all cluster nodes.' >&2
  exit 1
fi
```
This again applies to all `.slurm` files updated in this PR.